src/App.jsx

```diff
@@ -1,12 +1,88 @@
 import './App.css'
+import {useEffect, useState} from "react";
+import axios from "axios";
+import Card from "./components/Cards/Card.jsx";
+import PokemonLogo from './assets/Pokémon_logo_2.svg';
+
+// comment
 
 function App() {
 
-  return (
-    <>
-      <h1>Gotta catch em all!</h1>
-    </>
-  )
+    const [error, setError] = useState();
+    const [pokemons, setPokemons] = useState(null);
+    const [count, setCount] = useState(0);
+    const maxValue = 100;
+
+    useEffect(() => {
+        async function fetchData() {
+            try {
+                const response = await axios.get('https://pokeapi.co/api/v2/pokemon/');
+                console.log(response.data.results);
+                setPokemons(response.data.results);
+
+            } catch (e) {
+                console.error(e);
+                setError('Het ophalen van de data is mislukt!');
+
+            } finally {
+                console.log('Pokemons state after API call', pokemons);
+            }
+        }
+
+        fetchData();
+    }, []);
+
+    const handlePrev = () => {
+        if (count - 20 >= 0) {
+            setCount(count - 20);
+        }else{
+            setCount(0);
+        }
```

Comment:
Als je gebruikmaakt van het disabled attribuut, dan hoef je hier deze checks niet te doen
```suggestion
        setCount(count - 20);
```